tests/e2e/specs/in-game/basicMoves.spec.js

```diff
@@ -230,7 +229,6 @@ describe('Game Basic Moves - P0 Perspective', () => {
   });
 
   it('Plays Queens', () => {
-    cy.skipOnGameStateApi();
```

Comment:
Turns out this test can't be included yet because it requires playing a jack. Keep this as skipped